elements/lisk-transactions/README.md

```diff
@@ -2,6 +2,92 @@
+7. It is probable, especially shortly after a block is applied, that due to the decentralized network conditions a node does the `undo` step and the block containing all of the included transactions get reverted in favour of a competing block.
+
+While implementing a custom transaction, it is necessary to implement some of the mentioned steps. For most of them, a base transaction implements a default behaviour. As you feel more confident in using Lisk SDK, it is more likely for developers to override most of the base transaction methods, so the implementation is well-tailored and implemented with the best possible performance to the application's use case.
+
+### Interface
+
+Over the course of the alpha phase there will be significant changes in the Lisk protocol and implementation. We will be working towards the great developer experience while using Alpha SDK by reducing and optimizing the set of functions in custom transaction's API in the following releases. We strongly rely on the community feedback of how the interfaces can be improved.
+
+#### Required methods
+
+All of the abstract methods and properties on the base transaction's interface are required to implement. Those are:
+
+##### TYPE
+
+> static TYPE: number
+
+The hallmark of a transaction. Override this static parameter with any number, keeping in mind that the first 10 types (0-9) are reserved for the default transactions.
+
+##### applyAsset
+
+> applyAsset(store: StateStore): ReadonlyArray<TransactionError>
+
+The business use-case of a transaction is implemented in `applyAsset` method. Apply all of the necessary changes from the received transaction to the affected account(s) by calling `store.set`. Call `store.get` to get all of the relevant data. The transaction that you're currently processing is the function's context (like `this.amount`).
+Invalidate the transaction by pushing an error into the result array.
+
+##### undoAsset
+
+> undoAsset(store: StateStore): ReadonlyArray<TransactionError>
+
+The invert of `applyAsset`. Roll-back all of the changes to the accounts done in the `applyAsset` step.
+
+##### validateAsset
+
+> validateAsset(): ReadonlyArray<TransactionError>
+
+Before a transaction reaches the apply step it gets validated. Check the transaction's asset correctness from the schema perspective (no access to StateStore here).
+Invalidate the transaction by pushing an error into the result array.
+
+##### prepare
+
+> prepare(store: StateStorePrepare): Promise<void>
+
+Prepare the relevant information about the accounts, which will be accessible in the later steps during the `apply` and `undo` steps.
+
+#### Additional methods
+
+To increase your application's performance, you should override the following functions: `verifyAgainstTransactions`, `assetFromSync`, `fromSync`.
```

Comment:
Should we add a brief explanation for verifyAgainstTransactions somewhere?

Reply:
I think it's best to start from good descriptions in the code itself and generate docs in an automatic way.

Reply:
I added the GitHub Pages and now inserted the reference to [liskhq.github.io/lisk-sdk](https://liskhq.github.io/lisk-sdk/)

Reply:
I added the GitHub Pages and now inserted the reference to [liskhq.github.io/lisk-sdk](https://liskhq.github.io/lisk-sdk/)

Reply:
I added the GitHub Pages and now inserted the reference to [liskhq.github.io/lisk-sdk](https://liskhq.github.io/lisk-sdk/)